commands/changed/__tests__/__snapshots__/changed-command.test.js.snap

```diff
@@ -52,8 +52,8 @@ exports[`ChangedCommand circular should list changes with --force-publish * 1`]
 
 exports[`ChangedCommand circular should list changes with --force-publish package-2 1`] = `
 "- package-2
-- package-3
-- package-4"
+- package-4
```

Comment:
So this is probably the reason there's still a separate loop. This is incorrect output for `lerna changed`, which should always report packages with changes in lexical (alphabetical) order. We might as well remove the short-circuit early return, because a mere `forEach` over the `packages` Map is _not_ an expensive proposition.